.gitmodules

```
@@ -27,8 +27,8 @@
 #
 [submodule "fates"]
 path = src/fates
-url = https://github.com/NGEET/fates
-fxtag = sci.1.87.2_api.41.0.0
+url = https://github.com/rgknox/fates
+fxbranch = cnp-l2fr-conly
```

```suggestion
url = https://github.com/rgknox/fates
#fxtag = cnp-l2fr-conly     # This is the branch being pointed to, the hash used on the branch is below
fxtag = 8c5a24ce880ac117dffa7f5b87427a57250af8ed
```

@rgknox it looks like fxbranch isn't something that git-fleximod understands. It might have had it in the past, but doesn't now. When you run baselines I also recommend to point to either a tag or a hash so that you know exactly what you tested with. Naming a branch means the code at the tip of the branch may change after the tests are run.

So personally I usually have a comment that names what the branch is, and then use fxtag to point to the hash along the branch.

---

I'm going to go ahead and apply the above suggestion here, so that I can run Izumi tests that way.